src/modules/endpoint/reducers/__tests__/cityContentReducer.spec.js

```diff
@@ -85,7 +85,12 @@ describe('cityContentReducer', () => {
   ]
   for (const action of unsupportedActionsOnUnitializedState) {
     it(`should throw on ${action.type} if state is unitialized`, () => {
```

Comment:
Probably the test should be split up into the ones that throw and the ones that return null.
Then you can name the tests according to what the actions do.